screens/Issuers/IssuersScreen.tsx

```diff
@@ -124,10 +129,27 @@ export const IssuersScreen: React.FC<
     );
   }
 
+  const filterIssuers = text => {
+    let filterdData = issuers;
+    if (text.length > 0) {
```

Comment:
no need text.length here as we are not passing any arguments for default case where all issuers are listed.
we can use **if(text){} instead of if(text.length > 0)**

Reply:
If I follow this I need to pass an empty string into the respective auto-calling method(i.e., filterIssuers) through the onLayout prop of textInput component.